FindxDAQ.cmake

```diff
@@ -97,8 +97,9 @@ function(xdaq_import_lib name)
     find_library(
         xdaq_${name}_library
         ${name}
+        NO_CMAKE_ENVIRONMENT_PATH NO_CMAKE_SYSTEM_PATH
+        HINTS ENV XDAQ_ROOT
```

Comment:
In the documentation, it looks like a path is always required:
```
find_path (
          <VAR>
          name | NAMES name1 [name2 ...]
          [HINTS path1 [path2 ... ENV var]]
          ...
```

Reply:
Yeah, I know, but it seems to work. In the worst case, let just add an empty/dummy _path1_.